src/js/actions/authActions.js

```diff
@@ -0,0 +1,85 @@
+import fetch from 'fetch-everywhere';
+import {push} from 'react-router-redux';
+import urlFor from '../helpers/urlFor';
+
+const requestLogout = () => ({
```

Comment:
The methods I have don't mutate anything with the given input though. The actions are just dispatched and received by the reducers which return a new state and don't mutate previous state object.